public/content/docs/mels-hack/index.en.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "Mel's hack - the missing bits"
+author: "(David *)Frenkiel"
+date: Thu Jun 23 2022 01:14:31 GMT+0300
+---
+
+### A private _Mel moment_
+
+        When the light went on it nearly blinded me. 
+
```

Comment:
I would consider adding a short paragraph (even a mere mention) that this article is aimed for people who has some background in CS and are interested in Mel's actual hack.

Also, consider adding a link to the Preface article, for those who decide they want a "light light" version.